static/.eslint-ce-static.yml

```diff
@@ -27,9 +27,9 @@ rules:
     - error
     - smart
   indent:
-    - error
-    - 4
-    - SwitchCase: 1
+    - off
```

Comment:
Oh! What happened here?

Reply:
Ah, this is a remnant of #4775. There are some weird edge cases where prettier and eslint disagree on indentation, I turned off the rule for the back end in #4775 but not the front end since there wasn't a need yet :)